x-pack/plugins/observability/server/utils/queries.ts

```diff
@@ -29,7 +29,9 @@ export function rangeQuery(
         [field]: {
           gte: start,
           lte: end,
-          format: 'epoch_millis',
+          ...(typeof start === 'number' || typeof end === 'number'
```

Comment:
No need for `typeof`

Reply:
Need type check here because if either start or end is numeric then the additional `format` param is different. Either that or I can revert this change to use our old getRangeQuery instead https://github.com/elastic/kibana/pull/99905/commits/4686225257ec6ed458a15595911da8838a6c15d9#diff-4925538e11877e2f972a1ec1760ca92908c72a00da9e316493f21e6cb87d85d4L26